tests/virt_autotest/esxi_open_vm_tools.pm

```diff
@@ -137,16 +164,34 @@ sub get_epoch_time {
 sub init_guest_time {
     my $h_datetime = get_host_timestamp();
     # Get last day of the host time
-    my $last_day = script_output("date -u -d '$h_datetime last day' +'\%F \%T'");
+    my $last_day;
+    if (check_var('BACKEND', 'svirt')) {
+        $last_day = script_output("date -u -d '$h_datetime last day' +'\%F \%T'");
+    }
+    elsif (check_var('BACKEND', 'qemu')) {
+        $last_day = script_output(qq($ssh_vm "date -u -d '$h_datetime last day' +'\%F \%T'"));
```

Comment:
It seems this cmd can work for both backends, why differentiate?

Reply:
Yes, it doesn't work by some special characters with ssh which post a failure.